src/latexgen.cpp

```diff
@@ -294,8 +294,8 @@ static void writeLatexMakefile()
   }
   TextStream t(&f);
   // inserted by KONNO Akihisa <konno@researchers.jp> 2002-03-05
-  QCString latex_command = theTranslator->latexCommandName().quoted();
-  QCString mkidx_command = Config_getString(MAKEINDEX_CMD_NAME).quoted();
+  QCString latex_command = theTranslator->latexCommandName();
```

Comment:
I think this change will break things if an existing Doxyfile contains something like this:

```
LATEX_CMD=C:\Program Files\MiKTeX\miktex\bin\x64\latex.exe
```

Reply:
If adding `-enable-write18` is always required anyway when the image format SVG is used, then doxygen could also detect this and only then add the option itself, rather than relying on the user to do this.

Reply:
In principle yes but due the security risks I think it would be better to do it more explicitly and not implicitly, although for other parts doxygen also automatically starts sub programs, like `dot` of filters etc., so it should be possible to add it implicitly (and place a warning in the manual).